project_wbs_stage/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017-19 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017-19 Luxim d.o.o.
+# Copyright 2017-19 Matmoz d.o.o.
+# Copyright 2017-19 Deneroteam.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Project WBS Stage",
+    "version": "14.0.1.0.0",
+    "author": "Eficent,ForgeFlow",
+    "license": "LGPL-3",
```

Comment:
@JasminSForgeFlow please change license to AGPL-3 so it matches dependency

Reply:
done, thanks